src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -992,7 +994,10 @@ public async Task ReplaceClusterCert()
                             throw new InvalidOperationException("Cannot replace the server certificate. Only a ClusterAdmin can do this.");
 
                         var timeoutTask = TimeoutManager.WaitFor(TimeSpan.FromSeconds(60), ServerStore.ServerShutdown);
-
+                        if (Logger.IsOperationsEnabled)
+                        {
+                            Logger.Operations("Initiating the replacement of the certificate upon explicit request");
```

Comment:
more like an audit log? would also be good to add the requester then

Reply:
Should it be only in the audit log?
I moved the "source" of the certificate replacement outside of `StartCertificateReplicationAsync` function since here we know what the source is.
Before we log it as LetsEncrypt or executable (and sometimes it was wrong).
If I remove it from the log we will have this information missing from the logs.
I am addition the endpoint path